tests/e2e/policy_driven_vol_allocation.go

```diff
@@ -1374,6 +1377,635 @@ var _ = ginkgo.Describe("[vol-allocation] Policy driven volume space allocation
 		deletePodsAndWaitForVolsToDetach(ctx, client, newPods, true)
 
 	})
+
+	/*
+		verify volume allocation change post snapshot deletion works fine
+		Steps for detached volumes:
+		1	Create 2 SPBM policies with thin and LZT volume allocation respectively, say pol1, pol2
+		2	Create 2 SCs each with a SPBM policy created from step 1
+		3	Create a PVC of 10g using each of the SCs created from step 2
+		4	Wait for PVCs created in step 3 to be bound
+		5	Create a snapshot class
+		6	Create a snapshots of all PVCs created in step 3
+		7	Change volume allocation of pol1 and pol2 to EZT and apply the changes
+		8	Verify the volume allocation type changes fail
+		9	Delete snapshots created in step 6
+		10	Apply volume allocation type changes again and verify that it is successful this time
+		11	Delete the PVCs created in step 3
+		12	Delete the SCs created in step 2
+		13	Delete the SPBM policies created in step 1
+
+		Steps for attached volumes:
```

Comment:
Are we covering here for both online and offline support here? I think we should delete the pods if we are also covering for offline here